app/js/controllers.js

```diff
@@ -3537,7 +3537,7 @@ angular.module('myApp.controllers', ['myApp.i18n'])
     }
 
     $scope.$on('history_delete', function (e, historyUpdate) {
-      if (historyUpdate.msgs[$scope.messageID]) {
+      if (historyUpdate && historyUpdate.msgs && historyUpdate.msgs[$scope.messageID]) {
```

Comment:
This is the way most of the code checks null/empty values, so I preserved the style.